<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tt3::gui::CreateBeneficiaryDialog</class>
 <widget class="QDialog" name="tt3::gui::CreateBeneficiaryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create beneficiary</string>
  </property>
  <property name="windowIcon">
   <iconset resource="tt3-gui.qrc">
    <normaloff>:/tt3-gui/Resources/Images/Actions/CreateBeneficiaryLarge.png</normaloff>:/tt3-gui/Resources/Images/Actions/CreateBeneficiaryLarge.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="displayNameLabel">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="descriptionLabel">
     <property name="text">
      <string>Description:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QPlainTextEdit" name="descriptionPlainTextEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>85</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="workloadsLabel">
     <property name="text">
      <string>Workloads:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1" rowspan="2">
    <widget class="QListWidget" name="workloadsListWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>64</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>64</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>64</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="selectWorkloadsPushButton">
     <property name="text">
      <string>Select</string>
     </property>
     <property name="icon">
      <iconset resource="tt3-gui.qrc">
       <normaloff>:/tt3-gui/Resources/Images/Actions/ViewWorkStreamSmall.png</normaloff>:/tt3-gui/Resources/Images/Actions/ViewWorkStreamSmall.png</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>28</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="displayNameLineEdit"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>displayNameLineEdit</tabstop>
  <tabstop>descriptionPlainTextEdit</tabstop>
  <tabstop>workloadsListWidget</tabstop>
  <tabstop>selectWorkloadsPushButton</tabstop>
 </tabstops>
 <resources>
  <include location="tt3-gui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>tt3::gui::CreateBeneficiaryDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>tt3::gui::CreateBeneficiaryDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>displayNameLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>tt3::gui::CreateBeneficiaryDialog</receiver>
   <slot>_displayNameLineEditTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>descriptionPlainTextEdit</sender>
   <signal>textChanged()</signal>
   <receiver>tt3::gui::CreateBeneficiaryDialog</receiver>
   <slot>_descriptionPlainTextEditTextChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectWorkloadsPushButton</sender>
   <signal>clicked()</signal>
   <receiver>tt3::gui::CreateBeneficiaryDialog</receiver>
   <slot>_selectWorkloadsPushButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_displayNameLineEditTextChanged(QString)</slot>
  <slot>_descriptionPlainTextEditTextChanged()</slot>
  <slot>_selectWorkloadsPushButtonClicked()</slot>
 </slots>
</ui>
